.github/workflows/docs.yml

```diff
@@ -25,9 +25,9 @@ jobs:
 
       - name: Download Packages
         run: |
-          go install golang.org/x/tools/cmd/godoc@latest
-          go get github.com/johnstarich/go/gopages
-          go install github.com/johnstarich/go/gopages
+          go install golang.org/x/tools/cmd/godoc@v0.3.0
+          go get github.com/johnstarich/go/gopages@v0.1.11
```

Comment:
If you do `go install github.com/johnstarich/go/gopages@v0.1.11`
Then you don't need to `go get github.com/johnstarich/go/gopages@v0.1.11`

Reply:
Seems to be working, done 👍